src/Integration/Suppression/SuppressedIssuesProvider.cs

```diff
@@ -69,8 +69,15 @@ private static CreateProviderFunc GetCreateProviderFunc(ISonarQubeService sonarQ
             this.activeSolutionBoundTracker = activeSolutionBoundTracker ??
                                               throw new ArgumentNullException(nameof(activeSolutionBoundTracker));
 
+            if (activeSolutionBoundTracker.CurrentConfiguration == null)
+            {
+                throw new ArgumentNullException(nameof(activeSolutionBoundTracker.CurrentConfiguration));
```

Comment:
We get a code smell warning here, I'm not sure what to do with it.

Reply:
I don't think there is much reason to specifically check for a null configuration here - we don't check in line 88.

You could assert, but I'm not sure there's a lot of value in that either.